compiler/src/main/java/io/grpc/kotlin/generator/GrpcClientStubGenerator.kt

```diff
@@ -193,7 +199,7 @@ class GrpcClientStubGenerator(config: GeneratorConfig) : ServiceCodeGenerator(co
         %methodDescriptor:L,
         %parameter:N,
         callOptions,
-        %headers:T()
+        %headers:N
```

Comment:
potentially dumb question, buy why is the headers and parameters placeholder the same (`:N`)?